guides/_posts/2014-01-5-vfl.html

```diff
@@ -300,34 +301,144 @@
 
 {% highlight css %}
 /* VFL */
-@horizontal [#flexibleButton(>= 70, <= 100))];
+@h (#flexibleButton(>= 70, <= 100)));
 
 /* Equivalent CCSS */
 #flexibleButton[width] >= 70;
 #flexibleButton[width] <= 100;
 {% endhighlight %}
 
-<img class="vfl-small-img" src="../../assets/images/vfl/completeLayout.png" width="315" height="27" alt="CompleteLayout">
+
+## VFL Selectors
+All previous example of VFL expressions use basic [selectors](/guides/selectors). But you can also use combinators in VFL expression
+like this:
 
 {% highlight css %}
-/* VFL */
-@horizontal |-24-[#find]-[#findNext]-[#findField(>=30)]-24-| in(#popup);
+@v | (.featured > article ~ .title) | in(#elm > .class .class2"_fallout")
+{% endhighlight %}
+
+<a name="vflsplats"></a>
+## VFL Splat
+When a selector in your VFL expression finds more than one element, due to [plural binding](/guides/selectors/#pluralbinding),
+if the number of element within each selectors expression are not equal, then some elements will not be constraint.
+<a href="http://codepen.io/cbchouinard/pen/vEJqbG" target="_blank">Live example.</a>
+
+In this example, the last two `article` elements will not be constrained since there's no `section` element to bind to.
+As you can see in the CCSS generated constraints, the elements are pluraly binded.
```

Comment:
Suggestion:

> In this example, the last two `article` elements will not be constrained since there is no `section` element to bind to. As can be seen in the generated CCSS, the elements are constrained using plural binding.